usage.py

```diff
@@ -25,6 +33,11 @@
 def display_output(value):
     return 'You have entered {}'.format(value)
 
+@app.callback(Output('comparison', 'source'), [Input('input', 'value')])
+def update_comparison(value):
+    print(value)

```

Comment:
Was this debugging/testing code?  Looks useful, but might be worth cleaning it up.